packages/middleware-sdk-s3/src/throw-200-exceptions.spec.ts

```diff
@@ -15,6 +15,54 @@ describe("throw200ExceptionsMiddlewareOptions", () => {
     jest.clearAllMocks();
   });
 
+  describe("exceptions for code < 200 and >= 300", () => {
+    mockStreamCollector.mockResolvedValue(Buffer.from(""));
+    mockUtf8Encoder.mockReturnValue("");
+    it("throws an exception if code is less than 200", async () => {
```

Comment:
Lot of code is shared between the two tests, where only statusCode is different.
You can use [it.each](https://jestjs.io/docs/api#testeachtablename-fn-timeout) for it as follows:

```js
it.each([199, 300])("throws for status code %d", async (statusCode) => {
```